pandas/core/frame.py

```diff
@@ -4737,6 +4738,9 @@ def sort_values(
             indexer, axis=self._get_block_manager_axis(axis), verify=False
         )
 
+        if ignore_index:
+            new_data.axes[1] = ibase.default_index(len(indexer))
```

Comment:
Same comment as in other PR but this can just be `.drop` to match the Series implementation right?

Reply:
Ignore this comment - directly assign to `.index` as requested by Jeff

Reply:
emm, i think we should keep this as is. this `new_data` here is different than `result` in `series.py`, since here this is BlockManager, while in series, it is Series.

I used `.index` for the part in `series.py`!